docs/src/faqs/index.rst

```diff
@@ -27,6 +27,7 @@ FAQs
     custom_proto_def
     dags_in_klio
     migrate_from_fnapi
+    setup_packaging
```

Comment:
This just adds it to the left sidebar. You should also add the doc in one of the sections below (perhaps under "Technical").